clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java

```diff
@@ -62,10 +62,6 @@ class Builder {
 
     private Builder() {}
 
-    public URI getBaseUri() {
```

Comment:
This may have dependencies outside of Nessie. I do not think it's justified to remove this in a test refactoring PR :thinking:

Reply:
As it is a public interface and at least we should deprecate it before directly removing it.

Also, better to deprecate in separate PR and revert the change here!

Reply:
TBH, I'm not sure we should deprecate this :) Why not keep it as a convenience method?

Reply:
i think we removed the only user in https://github.com/projectnessie/nessie/pull/5354 but OK i have added it back